<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NamesPage</class>
 <widget class="QWidget" name="NamesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>Enter Player Names</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="names_scroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="names_container">
      <layout class="QVBoxLayout" name="names_layout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="contentsMargins">
        <rect>
         <x>12</x>
         <y>12</y>
         <width>12</width>
         <height>12</height>
        </rect>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="generate_button">
     <property name="text">
      <string>Generate Schedule</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="backLayout">
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="backSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
